docs/prms/oi2.rst

```diff
@@ -18,3 +18,12 @@ OI2 takes a few optional arguments:
     * "others" = connect to all nodes except for terminals
     * "all" = connect to all nodes in the interactome.
 * seed: The random seed to use for this run.
+
+
+Dataset Usage
+-------------
+
+OmicsIntegrator2 prefers ``prize``s, but will take the union of ``sources`` and ``targets``
+and set their 'prize' to 1 if ``prize`` is not specified.
+
+OmicsIntegrator2 does not consider graph directionality.
```

Comment:
This is confusing, it just treats an edge as undirected. Also should the directionality implementation be explained here as well (all directed edges are converted to undirected?